README.md

```diff
@@ -16,11 +25,11 @@ of the components to the developers, providing a single source of truth for both
 There is no such thing as out-dated and static PNG screens, as the current version of both the
 design models and the component implementation always render to up-to-date web pages instead.
 
-## Installation and usage
```

Comment:
There should be only one first level heading in every document

Reply:
Good point. Changed it and put the ToC after the intro